doc/getting_started/index.rst

```diff
@@ -230,27 +260,40 @@ additional Python dependencies.
             west zephyr-export
 
       #. Zephyr's ``scripts/requirements.txt`` file declares additional Python
-         dependencies. Install them with ``pip3``.
+         dependencies. Install them with ``pip`` into the virtual environment.
 
          .. code-block:: bash
 
-            pip3 install -r ~/zephyrproject/zephyr/scripts/requirements.txt
+            pip install -r ~/zephyrproject/zephyr/scripts/requirements.txt
 
    .. group-tab:: Windows
 
-      #. Install west:
+      #. Create and switch to a new virtual environment:
+
+         .. code-block:: console
+
+            python3 -m venv --prompt zephyrproject %HOMEPATH%\zephyrproject\venv
+            %HOMEPATH%\zephyrproject\venv\Scripts\activate.bat
+
+         When you've switched into the virtual environment, you'll see
+         ``(zephyrproject)`` at the beginning of your ``cmd.exe`` prompt:
+
+         .. code-block:: none
+
+            (zephyrproject) [...]>
+
+      #. Install west into the virtual environment:
 
          .. code-block:: bash
 
-            pip3 install west
+            pip install west
```

Comment:
I'd think this (and similar) changes are superfluous. pip3 should work in .venv, and it's better to rely on people's finger memory (and promote such) to use "pip3" next ~5 years, to not randomly fall on a case of python2.

Reply:
Fair enough. I am going to close this though since there is no interest in merging it for the main getting started guide. We can revisit if this is taken up again at some point in the future.

I'm not interested in rewriting this to live off to the side in some alley nobody will visit :).